generator/uscore/metadata_extractor.rb

```diff
@@ -206,11 +206,13 @@ def add_search_param_descriptions(profile_definition, sequence)
           profile_element = profile_definition['snapshot']['element'].select { |el| el['id'] == path }.first
           param_metadata = {
             path: path,
-            comparators: {}
+            comparators: {},
+            values: []
```

Comment:
Could this be a `Set` in case there are repeated codings from the different sources?

Reply:
fixed